webdriver-spec.html

```diff
@@ -3237,183 +3237,428 @@ <h2 id=element-displayedness>Element Displayedness</h2>
+
+            <li><p>Otherwise, return <var>cloned property result</var>.</li>
+          </ol>
+        </li>
+  </ol>
+
+  <p>The steps for <dfn>extracting the script arguments from a request</dfn>,
+  are:</p>
+   <ol>
+      <li><p>If the <a>current browsing context</a> is <a>no longer
+          open</a>, return an <a>error</a> with code <a>no such
+          window</a>.</p></li>
+
+      <li><p>Let <var>script</var> be the result of getting a property
+          named <code>script</code> from the parameters argument.</p></li>
+
+      <li><p>If <var>script</var> is not a <code>String</code> return
+          an <a>error</a> with code <a>invalid argument</a>.</p></li>
+
+      <li><p>Let <var>args</var> be the result of getting a property
+          named <code>args</code> from the parameters argument.</p></li>
+
+      <li><p>If <var>args</var> is not an <code>Object</code> or
+          its <code>[[\Class]]</code> internal property is
+          not <code>Array</code> or <code>Object</code>,
+          return <a>error</a> with code <a>invalid
+          argument</a>.</p></li>
+
+       <li><p>Let <var>arguments</var> be a <a>List</a> consisting of
+          a <a>json deserialization</a> of each item
+          in <var>args</var> with the order preserved.</p></li>
+
+       <li><p>Return <a>success</a> with data <var>script</var>
+       and <var>arguments</var></p></li>
+   </ol>
+
+   <p>When required to <dfn>execute a function body</dfn> with
+   arguments <var>body</var> and <var>arguments</var>, a <a>remote
+   end</a> must run the following steps:
+   <ol>
+      <li><p>Let <var>window</var> be the
+          <a href="https://html.spec.whatwg.org/#concept-document-window">associated
+            window</a> of the <a>current browsing
+            context</a>'s <a href="https://html.spec.whatwg.org/#active-document">active
+            document.</a></li>
+
+      <li><p>Let <var>environment settings</var> be the
+          <a href="https://html.spec.whatwg.org/#environment-settings-object">environment
+            settings object</a> for <var>window</var>.</p></li>
+
+      <li><p>Let <a>script environment</a> be
+          the <a href="https://html.spec.whatwg.org/#script-execution-environment">script
+          execution environment</a> for JavaScript obtained from
+          <var>environment settings</var>.</p></li>
+
+      <li><p>If <var>body</var> is not parsable as
+          a <a href="http://es5.github.io/#x13"><i>FunctionBody</i></a>
+          or if parsing detects an <i>early error</i>,
+          return <a>error</a> with code <a>javascript
+          exception</a>.</p></li>
+
+      <li><p>If <var>body</var> begins with
+          a <a href="http://es5.github.io/#x14.1">Directive
+          Prologue</a> that contains
+          a <a href="http://es5.github.io/#x14.1">Use Strict
+          Directive</a> then let <var>strict</var> be true, otherwise
+          let <var>strict</var> be false.</p></li>
+
```

Comment:
When evaluating a script in Marionette we need to say what the default JavaScript version is (see for example [Components.utils.evalInSandbox](https://developer.mozilla.org/en/docs/Components.utils.evalInSandbox)).  It feels we should say something about that as well.